src/schema/v1/artwork/index.ts

```diff
@@ -649,6 +671,16 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           return artwork.shipping_origin && artwork.shipping_origin.join(", ")
         },
       },
+      shippingCountry: {
+        type: GraphQLString,
+        description: "The country an artwork will be shipped from.",
+        resolve: artwork => {
+          return (
+            artwork.shipping_origin &&
+            artwork.shipping_origin[artwork.shipping_origin.length - 1]
```

Comment:
This worries me because we're making an assumption about the structure of an arbitrary array. Would probably be better to return a `shipping_origin_country` from gravity that we could use here, but given all of the logic is in this resolver I think we can follow up on that.